### PR TITLE
Updated readme

### LINKED ISSUE #143: Update repo ReadMe.md file
In line with the issue available [here](https://github.com/BHoM/admin/issues/4), please update your repositories ReadMe.md file to match the template provided in the linked admin issue.

Please remember to change any instances of `Robot_Toolkit` in the template to your repository name.

I will comment separately with a specific AppVeyor badge link that should be used for this repo.

When you resolve this issue, please add myself and @al-fisher as reviewers.

### PR DESCRIPTION
**Pull request by enarhi**
### Issues addressed by this PR

Closes #143

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,32 @@
-[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
-# BHoM
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/11a3ucotxcl9746k/branch/master?svg=true)](https://ci.appveyor.com/project/BHoMBot/ram-toolkit/branch/master) [![Build status](https://ci.appveyor.com/api/projects/status/r87fym5si9vr5io5/branch/master?svg=true)](https://ci.appveyor.com/api/projects/status/ram_toolkit/branch/master)
```
Comment (FraserGreenroyd) [inline README.md:1]: The first build status badge refers to Robot and should be replaced by the second badge which is in the [comment here](https://github.com/BHoM/RAM_Toolkit/issues/143#issuecomment-626582005).

There should also be the badge from Azure here as well.

**Comment by enarhi** [comment]
@FraserGreenroyd links have been updated.

File: README.md (hunk 1 above)
Comment (enarhi) [reply README.md]: Updated!

**Pull request merged**